system-tests/README.md

```diff
@@ -16,9 +16,10 @@ It uses Docker containers to create containerised instances of Kafka and other c
 
 * Run `pytest -s .` from the `system-tests/` directory
 
-* Wait
+* Optionally use `--local-build <PATH_TO_BUILD_DIR>` to run against a local build of the file writer rather than rebuilding in a docker container.
```

Comment:
```suggestion
* Optionally use `--local-build <PATH_TO_BUILD_DIR>` to run against a local build of the file writer rather than rebuilding in a docker container. Note that the build directory is the one containing the `bin` directory.
```
Minor clarification.